conf/minifi.properties

```diff
@@ -37,10 +37,37 @@ nifi.database.content.repository.directory.default=${MINIFI_HOME}/content_reposi
 #nifi.rest.api.user.name=admin
 #nifi.rest.api.password=password
 
+## Enabling C2 Uncomment each of the following options
+## define those with missing options
+#nifi.c2.enable=true
+## define protocol parameters
+#c2.agent.protocol.class=CoapProtocol
+#nifi.c2.agent.coap.host=
+#nifi.c2.agent.coap.port=
+#c2.rest.url=
+#c2.rest.url.ack=
```

Comment:
I understand we handle both c2.* and nifi.c2.* but can we migrate these for the sake of consistency?  Should be okay from the standpoint that we shouldn't break existing configs

Reply:
Oh shoot, good point. Thanks!